### PR TITLE
Layout updates

### PR DESCRIPTION
**Pull request by katrin-freihofner**

### CODE REVIEW

File: src/layout/columns.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+@media(min-width: $screen-md) {
+
+  .section {
+    display: flex;
+    justify-content: space-between;
+
+    .column--half {
+      flex: 0 0 50%;
+      max-width: calc(50% - 24px);
```
Comment (tomheller) [inline src/layout/columns.scss]: we could move the 24px to a variable?

Comment (tomheller) [inline src/layout/columns.scss]: Also, wouldn't it be more consistent to name this class `column--1-of-2`

Comment (tomheller) [inline src/layout/columns.scss]: Additionally, this could be done with some sass _magic_ and mixins?

File: src/layout/islands.scss [hunk 2]
```diff
@@ -28,5 +28,39 @@
...
 .layout.has-islands {
   padding: 12px;
-  //background-color: $gray160; TODO: Review
+}
+
+.island .content {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  max-width: 80pc;
```
Comment (tomheller) [inline src/layout/islands.scss]: why pc?

File: src/layout/columns.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  .column--2-of-3 {
+    max-width: 100%;
+  }
+}
+
+@media(min-width: $screen-md) {
+
+  .section {
+    display: flex;
+    justify-content: space-between;
```
Comment (lara-aigmueller) [inline src/layout/columns.scss]: We should add `flex-wrap: wrap`

File: src/layout/islands.scss [hunk 4]
```diff
@@ -28,5 +28,39 @@
...
+
+  .content--aside {
+    order: 0;
```
Comment (tomheller) [inline src/layout/islands.scss]: we might not need order: 0 here

File: src/layout/columns.scss (hunk 3 above)
Comment (tomheller) [inline src/layout/columns.scss]: Do we care about safari 8 here? I guess not.

File: src/layout/islands.scss [hunk 5]
```diff
@@ -28,5 +28,39 @@
...
+  }
+
+  .content--aside.on-top {
```
Comment (tomheller) [inline src/layout/islands.scss]: `.contentaside--top` or `content--aside--top`. choose wisely

File: src/layout/columns.scss (hunk 1 above)
Comment (lara-aigmueller) [inline src/layout/columns.scss]: `column--1-of-2` would be more consistent and `column--half` more intuitive. Maybe provide both options?

Comment (tomheller) [inline src/layout/columns.scss]: I'd rather have one definite way of naming the columns instead of providing a convenience feature.

**Comment by tomheller** [comment]
Awesome 🌈

**Pull request merged**